README.md

```diff
@@ -47,7 +47,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo](https://richardsmarioinfo.netlify.app/)
+- [Live Demo](https://richardmarioinfo.onrender.com/home)
```

Comment:
- [ ] After digging further, I discovered you added /home path to the url whereas, it should just be https://richardmarioinfo.onrender.com which is the root path. Kindly effect the change.